centreon-open-tickets/www/modules/centreon-open-tickets/views/rules/closeTicket/action.php

```diff
@@ -70,7 +73,7 @@ function isSelectionValid(string $selection): bool
 
 if (
     ! isset($data['rule_id'])
-    || ! is_int($data['rule_id'])
+    || ! is_int((int) $data['rule_id'])
```

Comment:
Hummm strange, use is_int() and the cas in same time is very strange :) you will have a integer all the time ;) You can remove the cast ;)

Reply:
ok